packages/web-frontend/cypress/scripts/generateConfig/generateReportConfig.js

```diff
@@ -123,10 +124,17 @@ const selectUrlParams = async (database, report, dashboardGroups) => {
     }
   }
 
-  return undefined;
+  // No compatible entry found, throw error
+  throw new Error(
```

Comment:
It's probably better to throw an error here than just skip this report, which is the existing functionality.

That way we'll be aware of the need to add new `orgUnitMaps.json` entries - which is a simple task really and shouldn't happen too often.